src/Neo/SmartContract/ApplicationEngine.Runtime.cs

```diff
@@ -452,6 +458,37 @@ protected internal void BurnGas(long datoshi)
             AddFee(datoshi * FeeFactor);
         }
 
+        /// <summary>
+        /// The implementation of System.Runtime.MintGas.
+        /// Minting GAS to benefit the current contract.
+        /// </summary>
+        /// <param name="amount">The amount of GAS to mint, in the unit of datoshi, 1 datoshi = 1e-8 GAS</param>
+        protected internal void MintGas(BigInteger amount)
```

Comment:
Technically, it's not a minted, as it will only be used in this script. It can be considered minted, but it will be claimed and burned

Reply:
I think the syscall should be renamed to "ChargeGas"